lib/mirage.ml

```diff
@@ -1912,6 +1912,8 @@ module Project = struct
     "open Lwt.Infix\n\
      let return = Lwt.return\n\
      let run = OS.Main.run"
+  let packages = [package "mirage"]
```

Comment:
I'm curious about this one... what kind of dependency are you expressing here?  that `Project` (as in, every MirageOS unikernel) depends on `mirage` at run time?  how comes that this is necessary?